.vscode/settings.json

```diff
@@ -84,6 +86,7 @@
         "Ævar"
     ],
     "cSpell.ignoreRegExpList": [
+        "CIimapHost",
```

Comment:
To be consistent, should the next few lines have upper camel casing to match this?

Reply:
I think those match the case of the instances with which cSpell had issues. I did not add them manually, but by hitting <kbd>Ctrl</kbd>+<kbd>.</kbd>, so I am fairly certain that the case is as intended.

Reply:
Guessing the case is ignored by cSpell.  The camelized case seems to be in the source.  I have vague recollections of playing with the names but did not see it in a quick search.  No bid deal.  Will continue review when I get a chance.

Reply:
Huh, you're right, the items _are_ case-insensitive, even in the `ignoreRegExpList` (I was under the impression that the regular expressions were more stringent, go figure).

In any case, I think the case of these patterns matches the text in the source code: https://github.com/gitgitgadget/gitgitgadget/blob/7499aa847f0dad77fb0cb556cc1ed512fadd7a8b/tests/ci-helper.test.ts#L22-L26 (note that `CIsmtpUser` does not need an exclusion because of the upper-case `U`).

> Will continue review when I get a chance.

Thank you!